.vib/suitecrm/vib-verify.json

```diff
@@ -3,7 +3,8 @@
     "resources": {
       "url": "{SHA_ARCHIVE}",
       "path": "{VIB_ENV_PATH}"
-    }
+    },
+    "runtime_parameters": "Y29tbWFuZDogWyJ0YWlsIiwgIi1mIiwgIi9kZXYvbnVsbCJd"
```

Comment:
Careful, you missed adding the `goss` action to the DSL and thus the test won't run.

Reply:
Good catch! Fixed now